src/api/app/controllers/build_controller.rb

```diff
@@ -168,12 +168,19 @@ def result
   def result_lastsuccess
     required_parameters :package, :pathproject
 
-    pkg = Package.get_by_project_and_name(params[:project], params[:package],
+    if params[:multibuild]
```

Comment:
Looking at the issue there is no multibuild parameter.

Other solution:

Set ``follow_multibuild:    true`` in Package.get_by_project_and_name will return always the package container. If the package parameter does not package pkg.name it is a multibuild.